perfkitbenchmarker/benchmark_sets.py

```diff
@@ -138,6 +138,10 @@
     'mit_set': {
         MESSAGE: 'Massachusetts Institute of Technology benchmark set.',
         BENCHMARK_LIST: [STANDARD_SET, 'silo']
+    },
+    'cloudsuite_set': {
+        MESSAGE: 'CloudSuite benchmark set.',
+        BENCHMARK_LIST: ['cloudsuite_inmemory_analytics']
```

Comment:
There should be 8 benchmarks eventually: Data Analytics, Data Caching, Data Serving, Graph Analytics, In-memory Analytics, Media Streaming, Web Search, Web Serving